server/sitemap.ts

```diff
@@ -4,6 +4,7 @@
 
 import { writeFileSync } from "fs";
 import { format } from "date-fns";
+import { findFirstLvlNav } from "../utils/general";
```

Comment:
can this be imported as `"utils/general"` or does it need to be relative?

Reply:
It does need to be relative. Because it is a sever part, with no aliases